modules/custom/az_publication/az_publication.routing.yml

```
@@ -2,6 +2,6 @@ az_publication.settings:
   path: '/admin/config/az-quickstart/settings/az-publication'
   defaults:
     _form: '\Drupal\az_publication\Form\AzPublicationSettingsForm'
-    _title: 'AZ Publication Settings'
+    _title: 'Publication'
```

I don't think we should change this.
```suggestion
    _title: 'AZ Publication Settings'
```